bokehjs/src/lib/models/ui/icons/tabler_icon.ts

```diff
@@ -0,0 +1,92 @@
+import {Icon, IconView} from "./icon"
+import {span, StyleSheet, ImportedStyleSheet, GlobalStyleSheet, StyleSheetLike} from "core/dom"
+import {isNumber} from "core/util/types"
+import * as p from "core/properties"
+
+export class TablerIconView extends IconView {
+  override model: TablerIcon
+
+  protected static readonly _url = "https://unpkg.com/@tabler/icons@latest/iconfont"
+
+  protected static readonly _fonts = new GlobalStyleSheet(`\
+    /*!
+    * Tabler Icons 1.68.0 by tabler - https://tabler.io
+    * License - https://github.com/tabler/tabler-icons/blob/master/LICENSE
+    */
+  @font-face {
+    font-family: "tabler-icons";
+    font-style: normal;
+    font-weight: 400;
+    src: url("${this._url}/fonts/tabler-icons.eot");
+    src: url("${this._url}/fonts/tabler-icons.eot?#iefix") format("embedded-opentype"),
+         url("${this._url}/fonts/tabler-icons.woff2") format("woff2"),
+         url("${this._url}/fonts/tabler-icons.woff") format("woff"),
+         url("${this._url}/fonts/tabler-icons.ttf") format("truetype"),
+         url("${this._url}/fonts/tabler-icons.svg#tabler-icons") format("svg");
```

Comment:
Sorry, dumb question. But these URLs are utilized at "view' time in the browser? If so we need to make it clear that these icons only work when the `unpkg.com` URL is network accessible.